test/auth/actions/use-unlocked-account-test.js

```diff
@@ -33,9 +33,10 @@ describe(`modules/auth/actions/use-unlocked-account.js`, () => {
       '../helpers/is-meta-mask': IsMetaMask,
     })
     sinon.stub(AugurJS.augur.rpc, 'isUnlocked').callsFake((address, callback) => {
-      t.stub.augur.rpc.isUnlocked(address, (isUnlocked) => {
-        store.dispatch({ type: 'AUGURJS_RPC_IS_UNLOCKED', data: { isUnlocked } })
-        callback(isUnlocked)
+      t.stub.augur.rpc.isUnlocked(address, (err, isUnlocked) => {
+        store.dispatch({ type: 'AUGURJS_RPC_IS_UNLOCKED', data: { isUnlocked: err || isUnlocked } })
+        if (err) return callback(err)
```

Comment:
Seems like this should also have a check like: 
```
if (isUnlocked == null) return callback('error calling augur.rpc.isUnlocked')
```

Reply:
`rpc.isUnlocked` cannot output a `null`.  If it outputs `undefined`, that should come back as an error!

Reply:
Yeah, you're right. Sorry -- there's no need to add this check.